pandas/core/groupby/grouper.py

```diff
@@ -100,7 +100,9 @@ def __new__(cls, *args, **kwargs):
             cls = TimeGrouper
         return super().__new__(cls)
 
-    def __init__(self, key=None, level=None, freq=None, axis=0, sort=False):
+    def __init__(
+        self, key=None, level=None, freq=None, axis=0, sort=False, dropna=True
```

Comment:
type what you are adding

Reply:
added

Reply:
emm, added and got the error below, i think this needs to have typing validation for this `Grouper` class

```
pandas/core/groupby/grouper.py:138: error: Incompatible types in assignment (expression has type "BaseGrouper", variable has type "None")
pandas/core/groupby/grouper.py:138: error: Value of type variable "FrameOrSeries" of "get_grouper" cannot be "None"
pandas/core/groupby/grouper.py:176: error: "None" has no attribute "take"
pandas/core/groupby/grouper.py:204: error: Incompatible types in assignment (expression has type "FrameOrSeries", variable has type "None")
```

Reply:
kk can do as a followon